src/sagemaker/model_monitor/dashboards.py

```diff
@@ -0,0 +1,353 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module contains code containing wrapper classes for dashboard structures in CloudWatch.
+
+These classes assist with creating dashboards in Python3 and then using boto3 CloudWatch client
+to publish the generated dashboards. To be used to aid dashboard creation in the create_monitoring_schedule
+and update_monitoring_schedule methods in model_monitoring.py
+"""
+
+import json
+
+
+class Variable:
```

Comment:
Is it possible to use a better name instead of Variable?

Reply:
I'll call it DashboardVariable, DashboardWidget, DashboardWidgetProperty to keep consistency.